content/spin/cli-reference.md

```diff
@@ -2860,65 +2907,11 @@ TRIGGER OPTIONS:
     -L, --log-dir <APP_LOG_DIR>
             Log directory for the stdout and stderr of components
 
-    -q, --quiet
-            Silence all component output to stdout/stderr
-
-        --runtime-config-file <RUNTIME_CONFIG_FILE>
-            Configuration file for config providers and wasmtime config
-            
-            [env: RUNTIME_CONFIG_FILE=]
-
-        --state-dir <STATE_DIR>
-            Set the application state directory path. This is used in the default locations for
-            logs, key value stores, etc.
-            
-            For local apps, this defaults to `.spin/` relative to the `spin.toml` file. For remote
-            apps, this has no default (unset). Passing an empty value forces the value to be unset.
-```
-
-{{ blockEnd }}
-
-{{ startTab "v1.1.0"}}
-
-<!-- @selectiveCpy -->
-
-```console
-$ spin up --help
-
-spin-up 
-Start the Spin application
-
-OPTIONS:
-        --direct-mounts         For local apps with directory mounts and no excluded files, mount
-                                them directly instead of using a temporary directory
-    -e, --env <ENV>             Pass an environment variable (key=value) to all components of the
-                                application
-    -f, --from <APPLICATION>    The application to run. This may be a manifest (spin.toml) file, a
-                                directory containing a spin.toml file, or a remote registry
-                                reference. If omitted, it defaults to "spin.toml"
-    -h, --help                  
-    -k, --insecure              Ignore server certificate errors from bindle server or registry
-        --temp <TMP>            Temporary directory for the static assets of the components
-
-TRIGGER OPTIONS:
-        --allow-transient-write
-            Set the static assets of the components in the temporary directory as writable
-
-        --cache <WASMTIME_CACHE_FILE>
-            Wasmtime cache configuration file
-            
-            [env: WASMTIME_CACHE_FILE=]
-
-        --disable-cache
-            Disable Wasmtime cache
+        --listen <ADDRESS>
```

Comment:
I thought the HTTP options were being shown in a separate subheading?

Reply:
That was a miss, fixed now.